views/js/qtiCreator/widgets/interactions/graphicGapMatchInteraction/states/Question.js

```diff
@@ -47,30 +47,32 @@ define([
 
     "use strict";
 
+    var GraphicGapMatchInteractionStateQuestion;
+
     /**
      * Media size runs not in automated mode, this applies the values manually
      *
      * @param params
-     * @param factor
      */
     function applyMediasizerValues(params, factor) {
-        factor = factor || 1;
-
         // css() + attr() for consistency
-        params.$target.css({
-            width: params.width * factor,
-            height: params.height * factor
-        })
-            .attr('width', params.width * factor)
-            .attr('height', params.height * factor);
+        params.$target
+            .css({
+                width: params.width * factor,
+                height: params.height * factor
```

Comment:
I don't understand the purpose here: your removed the jsdoc about `factor`, but left it in the function's params. Then you still use it in the CSS properties (without adjustment if undefined), but not in the attributes... Isn't there an inconsistency?